src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/NuGetUI.cs

```diff
@@ -365,15 +362,21 @@ public void ShowError(Exception ex)
             }
         }
 
-        private void ProcessSignatureIssues(PackageVerificationResult result, PackageIdentity packageIdentity)
+        private void ProcessSignatureIssues(SignatureException ex)
         {
-            var errorList = result.GetErrorIssues().ToList();
-            var warningList = result.GetWarningIssues().ToList();
+            UILogger.ReportError(ex.AsLogMessage().FormatWithCode());
```

Comment:
Why are you not checking here `if (!string.IsNullOrEmpty(ex.Message))` but you are doing this if in every other place where we log errors?